]fix: correct troubleshoot for failing host sca (#29637)
DataDog/documentation: content/en/security/code_security/troubleshooting/_index.md

The generated lock file is used by [`datadog-sbom-generator`][7] to extract dependencies and generate an SBOM.

> ### No Results from Datadog-hosted Scans

Reviewer: ```suggestion
### No results from Datadog-hosted scans
```

sentence-case

Datadog-hosted SCA scans do **not** support repositories that:

- Use [Git Large File Storage][18] (`git-lfs`)
- Contain invalid or reserved file paths (such as `/` or `\\`)
- Contain file paths with parent directory traversal (`..`)

> If your repository has any of these, and cannot be updated to account for those constraints, [set up the analysis in a CI pipeline][19] to run SCA and upload results to Datadog.

Reviewer: ```suggestion
If any of these conditions apply to your repository, and you cannot update your repository to account for these constraints, [set up the analysis in a CI pipeline][19] to run SCA and upload results to Datadog.
```
